docs/data/toolpad/building-ui/canvas-and-inspector.md

```diff
@@ -0,0 +1,61 @@
+# Canvas
+
+<p class="description">Canvas and Inspector are the two main areas when it comes to UI building</p>
+
+## Working with canvas
+
+To build your application's UI, drag the mouse over the component library to open the drawer. Click on a component and drag it over the canvas. The canvas will highlight the location where a component can be dropped. Release the mouse to place the component in the canvas.
+
+### Selecting components
+
+Click any component in the canvas to select it. An overlay indicates the selected components.
+
+<img src="/static/toolpad/docs/canvas-selected.png" width="232px" alt="Canvas controls" />
+
+### Moving components around
+
+In order to change position of a component select it first then click on the overlay handle, drag and drop it in the new location. The canvas will highlight available drop locations.
+
+<img src="/static/toolpad/docs/canvas-move.png" width="581px" alt="Canvas controls" />
+
+### Removing components
+
+Select the component you want to remove. Click the bin icon in the overlay to remove it from the canvas. You can also press the <kbd class="key">Backspace</kbd> key to achieve the same result.
+
+## Inspector
+
+In the inspector panel you can see the values of the selected component's properties. Each component has a specific set of properties that you can tweak in the inspector panel.
+
+<img src="/static/toolpad/docs/inspector.png" width="291px" alt="Inspector" />
+
+You can either use a static values or use dynamic ones as described in [data binding](/toolpad/data-binding/) section
+
+## Canvas grid
+
+### Rows
```

Comment:
`Rows` and `Columns` could be the same section, with a sentence like:
Toolpad canvas uses a grid layout where you can use rows to position components, or place them in a dedicated column besides other components.

Reply:
I'd keep it separate so we can use dedicated screenshots, but if we think that's not needed, we can also shorten it

Reply:
just a suggestion, feel free to do what you think is best!